app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/single_disposal_task_list.scala.html

```diff
@@ -192,7 +192,7 @@ <h2 class="app-task-list__section">
             </li>
 
             <li id="uploadSupportingDocs" class="app-task-list__item">
-                @taskLink(messages("task-list.upload-supporting-docs.link"), returns.uploadsupportingdocs.routes.UploadSupportingDocumentsController.hasSupportingDocsToUpload().url, uploadSupportingDocsState)
+                @taskLink(messages("task-list.upload-supporting-docs.link"), returns.uploadsupportingdocs.routes.UploadSupportingEvidenceController.checkYourAnswers().url, uploadSupportingDocsState)
```

Comment:
can you stick the file upload section to the multiple disposals task list as well?

Reply:
I think you just did this. Pls confirm

Reply:
I did it yeah, this is in both journeys now with the status of the big CYA section fixed up as well